pkg/controller/resourceactuation/resourceactuation.go

```diff
@@ -15,14 +15,86 @@
 package resourceactuation
 
 import (
+	"context"
 	"fmt"
+	"strings"
 
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/reconciliationinterval"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/k8s"
 
+	opv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/operator/pkg/apis/core/v1beta1"
+	opk8s "github.com/GoogleCloudPlatform/k8s-config-connector/operator/pkg/k8s"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// todo acpana: revisit
```

Comment:
Yeah, I don't think you need any of these here, I would just return the enum values and switch on them.  It's fine to be case-sensitive for enum values.

Reply:
n/a after adding enum values to types